langchain/src/callbacks/base.ts

```diff
@@ -510,6 +510,12 @@ export class CallbackManager extends BaseCallbackManager {
     return manager;
   }
 
+  copy(): CallbackManager {
+    const manager = new CallbackManager(this._parentRunId);
+    manager.setHandlers(this.handlers);
```

Comment:
This assumes handlers are stateless which isn't true eg of the tracer. Is the tracer "stateless enough" now that this can never be a problem?

Reply:
yup, this should not be a problem, no more stack. it uses a map in instead which will prevent the "collisions" problem